### DISCUSSION TITLE
Getting experimentalInteractiveRunEvents  to work

### DISCUSSION THREAD

**Discussion by flowt-au** [cypress-io/cypress#16541] [Questions and Help] [answered]
Hi. I am experimenting with Cypress and have the `experimentalInteractiveRunEvents`  flag ENABLED according to Settings / Experiments. 
I have this in my cypress\plugins\index.js file:
```js
    module.exports = (on, config) => {
      on('file:preprocessor', cucumber())  // I am also using cypress-cucumber

      on('after:spec', (spec, results) => {
        console.log('Finished running', spec.relative)
      })
    }
```
as per the docs but am not seeing the message in my browser console. Am I misunderstanding something here?

I am using v7.3.0

Thanks.

**Comment by bahmutov** [comment, marked as answer]
Have you checked the terminal? The plug-in file runs in the Node process, not in the browser, so those messages would be in the terminal where you launched Cypress

**Reply by flowt-au** [reply]
Doh! Of course. Sorry. yes, displaying now. 
Thanks,
Murray
